hms-app/src/components/card- details/HackathonDetails.tsx

```diff
@@ -361,6 +362,19 @@ export default function HackathonDetails(props: IProps) {
                   </Text>
                 }
               >
+                <Button
+                  mb={20}
+                  onClick={() =>
+                    localStorage.setItem(
```

Comment:
nice!

Reply:
however I would say to push this somewhere else.  Likely it should only ever be done by the `actions` portion of the frontend.  That should reduce a bit of code duplication that could pop up if multiple places can load ideas from the backend.  In fact it should probably happen something like this

1. ask some action function for ideas
2. check if they are in localstorage
  a. if they are use what is in localstorage and don't call the backend
  b. if not then call the backend to get the ideas
    I. store the ideas in localstorage
3. return the ideas to the caller

then we should have a boolean that can be passed in (set to false if nothing is passed in) that can force reloading of ideas (so it just skips checking in localstorage).

this could be done for every loadXxx() function